ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/workers-rhcos-rhel8/sdn/provision/cucushift-installer-rehearse-aws-ipi-disconnected-private-workers-rhcos-rhel8-sdn-provision-chain.metadata.json

```diff
@@ -0,0 +1,21 @@
+{
+	"path": "cucushift/installer/rehearse/aws/ipi/disconnected/private/workers-rhcos-rhel8/sdn/provision/cucushift-installer-rehearse-aws-ipi-disconnected-private-workers-rhcos-rhel8-sdn-provision-chain.yaml",
```

Comment:
Though `cucushift/installer/rehearse/aws/ipi/disconnected/private/workers-rhcos-rhel8/sdn/provision/` and `cucushift/installer/rehearse/aws/ipi/disconnected/private/sdn/workers-rhcos-rhel8/provision/` should have the same effect, for keeping consistent folder layout, do you mind update the layout a bit to `cucushift/installer/rehearse/aws/ipi/disconnected/private/sdn/workers-rhcos-rhel8/provision/`

Reply:
Sure, will update it.